save-cloud-common/src/commonMain/kotlin/org/cqfn/save/info/UserInfo.kt

```diff
@@ -22,6 +22,7 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class UserInfo(
     val name: String,
+    val password: String? = null,
```

Comment:
Actually, this is a good point. We shouldn't be constantly sending such sensitive information back and forth. Github, for example, shows the token only after its very creation and then allows only revoking and regenerating

Reply:
Right, we need to transfer token to backend and DB, so I add additional field into the class, which we use for transferring

Reply:
> Actually, this is a good point. We shouldn't be constantly sending such sensitive information back and forth. Github, for example, shows the token only after its very creation and then allows only revoking and regenerating

What do you mean about `back and forth`? We use it only once too. After new button submition it will generate new token, which we will save over existing one in DB

Reply:
Won't it be read from the DB into a `User` object and then filled in `UserInfo` every time we query for example `/sec/user`?

Reply:
I don't really think, that it's critical, but ok